examples/step-5/step-5.cc

```diff
@@ -113,6 +113,8 @@ double coefficient (const Point<dim> &p)
   else
     return 1;
 }
+// @note Mapping of this representation of the coefficient to the mesh cell
+// is non-trivial, see Results section of step-6 for more discussion.
```

Comment:
I would remove this. It is something that at this point isn't on the mind of the reader, and without further explanation is going to raise more questions than provide answers to readers.